t/50commit.t

```diff
@@ -59,7 +59,6 @@ EOT
 
   $dbh->{AutoCommit} = 0;
   ok !$dbh->err;
-  ok !$dbh->errstr;
```

Comment:
This no longer checks for no warnings, is it OK?

Reply:
Changing `$dbh->{AutoCommit}` does not produce warnings (or yes?).
If changing `$dbh->{AutoCommit}` fails then it set err (and raise an exception if RaiseError is enabled).

Reply:
OK, I checked the documentation of DBI (and DBD::MariaDB) and changing AutoCommit should never warn.